  <launch>

	<arg name="calibrated" 	default="true"/>
	
	<!-- Openni for head asus -->
	<include file="$(find bender_sensors)/launch/rgbd/asus/asus.launch">
		<arg name="asus_name"   value="rgbd_head"              />
		<arg name="device_id"   value="$(env BENDER_RGBD_HEAD_ID)"/>
		<arg name="publish_tf"  value="false"/>
		<arg name="calibrated" 	value="$(arg calibrated)"/>
	</include>
  
  <node name="detect_person"        pkg="basic_perception"      type="detection.py" output="screen">
    </node>

</launch>